backend/routes/social.route.ts

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+import * as socialController from '../controllers/social.controller';
+
+const router = express.Router();
+
+router.get('/', socialController.getAllSocials);
+router.get('/:id', socialController.getOne);
+router.post('/', socialController.createSocial);
+router.put('/:id', socialController.updateSocial);
```

Comment:
Router method supports chaining, so you can replace current version with:

```suggestion
router
  .get('/', Controller.getAllSocials)
  .post('/', Controller.createSocial)
  
  .get('/:id', Controller.getOne)
  .put('/:id', Controller.updateSocial)
```